web/components/admin/Toolbar/index.js

```diff
@@ -67,6 +85,19 @@ class Toolbar extends React.Component {
             src={areFiltersApplied ? filterIconSelected : filterIcon}
             alt="filter button" />
         </div>
+        <TextInput
+          name="search-applicants"
+          className="pad-left-xxl"
+          placeholder="Search..."
+          onChange={(searchText) => {
+            searchApplicants(searchText);
+          }} />
+        <Select
```

Comment:
About the label misalignment issue that you mentioned in your PR body, the width of the `.toolbar .select-input` is set to `265px` which isn't enough space to hold the label + dropdown. If you change that value to `auto` (in the index.sass file for the Toolbar), it should fix it.

Reply:
@chamkank okay so this actually fixed an error I hadn't noticed with the new select input! In my PR message though I meant that the search field/textbox is not vertically aligned, it's like a millimeter to low LOL

![screen shot 2019-01-22 at 9 46 21 pm](https://user-images.githubusercontent.com/39585481/51585687-30d60600-1e8f-11e9-9e11-0d1699c8791b.png)

Reply:
OHHH gotcha, I think the search box can be fixed with https://github.com/nwplus/nwhacks2019/pull/244#discussion_r249742497

Reply:
also I'm trying to make the toolbar more like the mocks, not sure if we'll end up keeping this but this is what the toolbar looks like when i add a search emoji as the label
![screen shot 2019-01-22 at 9 52 08 pm](https://user-images.githubusercontent.com/39585481/51585917-19e3e380-1e90-11e9-8802-2b6ebdd0a7e9.png)

Reply:
SORRY I didn't see your other comment, got the vertical alignment fixed 👍 The emoji search as label didn't work (even with the change and also trying width: auto it looked like the photo above), it was kind of dumb anyway so I'm just going to leave it as is

Reply:
Yeah doesn't seem like it adds any functionality, the search box by itself seems fine!